4-WebApp-your-API/4-1-MyOrg/README.md

```diff
@@ -1,225 +1,269 @@
 ---
-page_type: sample
-languages:
-  - csharp
-products:
-  - aspnet-core
-  - azure-active-directory  
-name: How to secure a Web API built with ASP.NET Core using the Microsoft identity platform (formerly Azure Active Directory for developers)
-urlFragment: active-directory-aspnetcore-webapp-openidconnect-v2
-description: "This sample demonstrates a ASP.NET Core Web App calling a ASP.NET Core Web API that is secured using Azure AD"
+services: active-directory
+platforms: dotnet
+author: v-abeyderman
+level: 200
+client: ASP.NET Core Web App
+endpoint: AAD v2.0
```

Comment:
also this is redundant

Reply:
I've removed it from template, do we want to cleanup sample.json as well?